CONFIGURATION.md

```diff
@@ -0,0 +1,72 @@
+# Configuration
+
+Document Merge Service is a [12factor app](https://12factor.net/) which means that configuration is stored in environment variables.
+Different environment variable types are explained at [django-environ](https://github.com/joke2k/django-environ#supported-types).
+
+## Common
+
+* `SECRET_KEY`: A secret key used for cryptography. This needs to be a random string of a certain length. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-SECRET_KEY).
+* `ALLOWED_HOSTS`: A list of hosts/domains your service will be served from. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts).
+
+## Database
+
+Per default [Sqlite3](https://sqlite.org/) is used as database for simple deployment and stored at `/var/lib/document-merge-service/data/sqlite3.db`. Create a volume to make it persistent.
+
+To scale the service a different database storage is needed. Any database supported by [Django](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-DATABASE-ENGINE) can be used.
+
+* `DATABASE_ENGINE`: Database backend to use.
+* `DATABASE_HOST`: Host to use when connecting to database
+* `DATABASE_PORT`: Port to use when connecting to database
+* `DATABASE_NAME`: Name of database to use
+* `DATABASE_USER`: Username to use when connecting to the database
+* `DATABASE_PASSWORD`: Password to use when connecting to database
+
+## Unoconv
+
+* `UNOCONV_ALLOWED_TYPES`: list of types allowed to convert to. See `unoconv --show` (default: ['pdf'])
+* `UNOCONV_SERVER`: hostname or ip of unoconv listener. If no server is specified a local unoconv
+   listener inside the container is used. Defaults to `None`.
+* `UNOCONV_PORT`: port of unoconv listener. Only relevant if `UNOCONV_SERVER` is set. Defaults to `2002`.
```

Comment:
These settings no longer exist. Checkout the `README.md`. 

(Another small nitpick is that we mix the capitalization of the list items. Maybe it would make sense to unify them?)